regulations/static/regulations/js/source/models/meta-model.js

```diff
@@ -51,73 +51,29 @@ var MetaModel = Backbone.Model.extend({
         return section;
     },
 
-    // **Param**
-    // id, string, dash-delimited reg entity id
-    //
-    // **Returns** boolean
     has: function(id) {
-        return (this.content[id]) ? true : false;
+      return !!this.content[id];
     },
 
-    // **Params**
-    //
-    // * ```id```: string, dash-delimited reg entity id
-    // * ```callback```: function to be called once content is loaded
-    get: function(id, callback) {
-        var $promise, resolve;
-
-        // if we have the requested content cached, retrieve it
-        // otherwise, we need to ask the server for it
-        $promise = (this.has(id)) ? this.retrieve(id) : this.request(id);
-
-        // callback once the promise resolves
-        resolve = function(response) {
-            if (typeof callback !== 'undefined') {
-                callback(true, response);
-            }
-        };
-
-        $promise.done(resolve);
-
-        $promise.fail(function() {
-            callback(false);
-        });
-
-        return this;
+    get: function(id, options) {
+      return this.has(id) ? this.retrieve(id) : this.request(id, options);
     },
 
-    // basically returns ```this.content[id]``` immediately,
-    // but is consistent with the interface that ```this.request``` provides
     retrieve: function(id) {
-        var $deferred = $.Deferred();
-
-        $deferred.resolve(this.content[id]);
-
-        return $deferred.promise();
+      return $.when(this.content[id]);
```

Comment:
In the final pass, would you mind documenting this function? Something like: "The contents are already cached, so return them as a fulfilled promise"

Kinda related, would it make sense to either rename the function or just inline it? `retrieve()` doesn't convey enough information and could easily be confused with `request()`